xbmc/video/VideoInfoScanner.cpp

```diff
@@ -682,7 +682,8 @@ CVideoInfoScanner::CVideoInfoScanner()
 
         if (fetchEpisodes)
         {
-          InfoRet ret = RetrieveInfoForEpisodes(pItem, lResult, info2, useLocal, pDlgProgress);
+          InfoRet ret =
+              RetrieveInfoForEpisodes(pItem, lResult, info2, useLocal, pDlgProgress, true);
```

Comment:
I know I'm late to the party, but from only reading this line, who could tell what this newly introduced `true`means? That's why I'm trying to avoid bool params, knowing that introducing an enum class to model the two possible values is a bit more work, but produces far more readable code. @78andyp